App/Config/DebugConfig.ts

```diff
@@ -12,7 +12,7 @@ const config: IDebugConfig = {
   showDevScreens: __DEV__,
   useFixtures: false,
   ezLogin: false,
-  yellowBox: __DEV__,
+  yellowBox: false,
```

Comment:
so much nicer to develop without this... fine to revert but seems like if you want it you could toggle it on instead of having to hit 'dismiss' every time you refresh the UI

Reply:
firebase added a few more lines of yellow warnings here too, maybe worth signing off on @asutula